test/src/api/commands/testWaitForElementVisible.js

```diff
@@ -163,7 +163,7 @@ describe('waitForElementVisible', function() {
     let result;
 
     this.client.api.globals.waitForConditionPollInterval = 5;
-    this.client.api.waitForElementVisible('.weblogin', 11, function (res, instance) {
+    this.client.api.waitForElementVisible({selector: '.weblogin', timeout: 11, suppressNotFoundErrors: false}, function (res, instance) {
```

Comment:
this shouldn't change. you can add an additional test for this check.

Reply:
Cool

Reply:
done